gum/backend-darwin/gumprocess-darwin.c

```diff
@@ -169,11 +169,80 @@ struct _DyldImageInfo64
   guint64 image_file_mod_date;
 };
 
-#ifndef PROC_SETPC_NONE
-extern int proc_regionfilename (int pid, uint64_t address, void * buffer,
-    uint32_t buffersize);
+#ifndef PROC_INFO_CALL_PIDINFO
+
+#define PROC_INFO_CALL_PIDINFO 0x2
+#define PROC_PIDREGIONPATHINFO 8
+
+struct vinfo_stat {
+  uint32_t vst_dev;    /* [XSI] ID of device containing file */
+  uint16_t vst_mode;   /* [XSI] Mode of file (see below) */
+  uint16_t vst_nlink;  /* [XSI] Number of hard links */
+  uint64_t vst_ino;    /* [XSI] File serial number */
+  uid_t vst_uid;    /* [XSI] User ID of the file */
+  gid_t vst_gid;    /* [XSI] Group ID of the file */
+  int64_t vst_atime;  /* [XSI] Time of last access */
+  int64_t vst_atimensec;  /* nsec of last access */
+  int64_t vst_mtime;  /* [XSI] Last data modification time */
+  int64_t vst_mtimensec;  /* last data modification nsec */
+  int64_t vst_ctime;  /* [XSI] Time of last status change */
+  int64_t vst_ctimensec;  /* nsec of last status change */
+  int64_t vst_birthtime;  /*  File creation time(birth)  */
+  int64_t vst_birthtimensec;  /* nsec of File creation time */
+  off_t vst_size;   /* [XSI] file size, in bytes */
+  int64_t vst_blocks; /* [XSI] blocks allocated for file */
+  int32_t vst_blksize;    /* [XSI] optimal blocksize for I/O */
+  uint32_t vst_flags;  /* user defined flags for file */
+  uint32_t vst_gen;    /* file generation number */
+  uint32_t vst_rdev;   /* [XSI] Device ID */
+  int64_t vst_qspare[2];  /* RESERVED: DO NOT USE! */
```

Comment:
How about dropping the comments? (And leave it up to the original header to explain.)